src/app/globals.css

```
@@ -181,3 +181,19 @@ h6 {
 		transform: translateX(0);
 	}
 }
+
+@keyframes fadeInUp {
```

Stylelint (17.6.0)

[error] Expected keyframe name "fadeInUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)